pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -258,6 +258,10 @@ def input(self, args, key):
                         # Wait for storage.
                         threadMgr.wait(THREAD_STORAGE)
 
+                        # Allow to format DASDs.
+                        self._disk_init_module.SetFormatUnrecognizedEnabled(True)
```

Comment:
Isn't this a bit dangerous? What about other disks which are not DASD, wouldn't this call enable us to destroy them too?

Reply:
Hmm, it looks like that. The partitioning module will initialize all disks with unrecognized formats. However, we already do the same in GUI and it was implemented this way from the beginning. The fix of this behaviour won't be trivial, so I suggest to go ahead with this solution and implement the proper one in upstream. What do you think?

Reply:
What would be these other unrecognized formats?

Reply:
I guess that every format that isn't supported by blivet? It is a DBus alternative to the kickstart command `zerombr`. This issue can happen only on 390x, so that might be the reason why it is not reported.

Reply:
So this is set to bite us in the future, once something (not-yet-)supported appears? Maybe extending the comment could help.

Reply:
Could you please add a comment with FIXME here?

Reply:
Fixed.